CPT331.WebAPI/Controllers/EventController.cs

```diff
@@ -23,11 +25,19 @@ public class EventController : ApiController
 
 		[HttpGet]
 		[Route("api/Event/{id}")]
-		public EventModel Event(int id)
+		public EventModel Event(uint id)
```

Comment:
I'm actually in two minds about this. Whilst uint enforces positive ints only, it feels a bit like finessing. I wonder if it's better to leave it as int and instead whack validation on there?